eng/scripts/Language-Settings.ps1

```diff
@@ -96,7 +96,8 @@ function Publish-javascript-GithubIODocs ()
     {
       $DocVersion = $dirList[0].Name
       Write-Host "Uploading Doc for $($PkgName) Version:- $($DocVersion)..."
-      Upload-Blobs -DocDir "$($DocLocation)/documentation/$($Item.BaseName)/$($Item.BaseName)/$($DocVersion)" -PkgName $PkgName -DocVersion $DocVersion
```

Comment:
How did you catch these updates? 

I'm only wondering because I think you also need to update the PackageInfoFromPackageFile  function to add the ReleaseTag. Similarly in the other languages as well.

Reply:
You are right. But for now am not calling the `PackageInfoFromPackageFile`  in the files. When I switch over to use those I will update it.